spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -106,15 +106,15 @@ def save_to_s3(self, saveDat, filepath, compress):
         key = ("/".join(path[1:])).lstrip("/") + ".json"
 
         if compress is False:
-            s3.put_object(Bucket=bucket, Body=json.dumps(saveDat).encode("utf-8"), Key=key)
+            s3.put_object(Bucket=bucket, Body=orjson.dumps(saveDat), Key=key)
         else:
-            dat = gzip.compress(json.dumps(saveDat).encode("utf-8"))
+            dat = gzip.compress(orjson.dumps(saveDat))
```

Comment:
We don't need to encode these since gzip just wants the bytes, right?

Reply:
The previous `encode` calls were actually taking the string returned by `json.dumps` and converting it to `bytes`